tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -96,27 +117,149 @@ def test_region_without_t2micro(vpc_stack, pcluster_config_reader, key_name, reg
     In other words, t3.micro is retrieved when the region does not contain t2.micro
     """
     config_path = test_datadir / "config.yaml"
-    stages = orchestrate_pcluster_configure_stages(
-        region,
+    region_without_t2micro_prompts = (
```

Comment:
Wy do we need to test region with absence of t2.micro?

Reply:
This is from a test that was already part of the suite. I only refactored it.
The test is meant to confirm that the default instance-type is retrieved dynamically. E.g. For regions that don't have `t2.micro`, the default instance-type is `t3.micro`.

Reply:
Got it thanks.